test/test_helper.exs

```diff
@@ -17,6 +17,7 @@ options = if version < {3, 4, 0} do [mongo_3_4: true] ++ options else options en
 ExUnit.configure exclude: options
 ExUnit.start()
 
+### currently disabled
```

Comment:
What exactly is disabled?

Reply:
I removed the comment. I started the test with a local mongodb instance and comment some lines in the test_helper.exs.